spec/models/notification_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+
+RSpec.describe Notification, type: :model do
+  let!(:super_admin) { create(:user, :super_admin)}
+  let!(:organization) { create(:organization) }
+  let!(:admin) { create(:user, :admin, organization_id: organization.id) }
```

Comment:
prefer to pass instance instead of id like `create(:user, :admin, organization: organization)`